### ISSUE TITLE
Image alt attributes

### ISSUE THREAD

**Issue by ghost** [Newsociety/prj-rev-bwfs-tea-cozy#4]
https://github.com/Newsociety/prj-rev-bwfs-tea-cozy/blob/master/Tea%20Cozy/index.html#L12

Give your `img` tags `alt` attributes that describe your images.

This improves your website's accessibility at times when images are unable to be rendered. Like, when visually impaired users use your website on devices or readers that don't render images.
